tests/mxnet/test_specialization.py

```diff
@@ -227,7 +227,7 @@ def message_func_edge(edges):
     def reduce_func(nodes):
         return {'v2': mx.nd.sum(nodes.mailbox['m2'], axis=1)}
 
-    g = generate_graph(100)
+    g = generate_graph(10)
```

Comment:
original size is too large?

Reply:
Oh, I change it to debug. I could change it back if you want.

Reply:
yes, let's change it back.

Reply:
done